src/components/Sections/PopularSection/PopularSection.tsx

```diff
@@ -23,7 +30,7 @@ export const PopularSection: React.FC = () => {
               alt={game.name}
               title={game.name}
               genre={game.genres[0].name}
-              rating={`${game.metacritic}%`} />
+              rating={game.metacritic ? `${game.metacritic}%` : ''} />
```

Comment:
and here you can just pass game.metacritic :D if you update form my previous comment

Reply:
this way you make sure that the PopularGame component is always handling in case there is no value.